src/erlcloud_sns.erl

```diff
@@ -495,6 +496,24 @@ set_topic_attributes(AttributeName, AttributeValue, TopicArn, Config)
             {"TopicArn", TopicArn}]).
 
 
+-spec(get_topic_attributes/1 :: (string()) -> ok).
+get_topic_attributes(TopicArn) ->
+    get_topic_attributes(TopicArn, default_config()).
+
+-spec(get_topic_attributes/2 :: (string(), aws_config()) -> ok).
+get_topic_attributes(TopicArn, Config)
+    when is_record(Config, aws_config) ->
+    Params = [{"TopicArn", TopicArn}],
+    Doc = sns_xml_request(Config, "GetTopicAttributes", Params),
```

Comment:
well. looks like this is doen really abnormal way in sns module https://github.com/andrey-smirnov/erlcloud/blob/c59c12990e9cddee63fe41b94f2add4faf29d06b/src/erlcloud_sns.erl#L641-L647 so that's is fine to be inline with the rest of the module. while it still has to be improved